### DISCUSSION TITLE
Doing differential analysis for multiomics data sets

### DISCUSSION THREAD

**Discussion by kendyhoangucsc** [stuart-lab/signac#947] [Q&A]
Hello,

I am currently working with three datasets, both snATAC and snRNAseq were sequenced from wild-type, heterozygous, and conditional knockout tissue. The way I went about processing this, was processing the different genotypes separately, performed the weighted nearest neighbor analysis, and annotated the clusters based off of gene expression and looking at the atac peaks of marker genes to see whether my annotation made sense. 

I was wondering if I wanted to do some differential analysis (for gene expression or for chromatin accessibility) between clusters in the different genotypes, how should I go about this?  

Would I have to merge the three datasets to accomplish something like this? or would it be possibly to get the barcodes from the cluster that I'm interested in let's say the WT and the similar cluster that I identified in let's say the mutant, merge them into a new seurat object and compare/run some differential analysis test between these two populations from the different genotypes.

How worried should I be if I process these these three genotype separately and how should I correct for batch effects? 

Thanks in advance

**Comment by timoast** [comment]
If you want to compare across genotypes then you'll need to have cells from each genotype in the same Seurat object, so you could merge them if that's your goal. If you just want to look at DE across clusters within a genotype there's no need to merge them.

> how should I correct for batch effects

You can include batch as a latent variable in the DE model (as long as you have replicates for each condition)

**Reply by kendyhoangucsc** [reply]
if I preprocessed all three genotypes and have the cluster's from each genotype annotated or gave them an identity, once I merge the three datasets together, can I perform differential analysis that way where the cells in this condition were named "X_WT" and in another condition "X_HET", where X in were the cluster of cells identified in each genotype. 

I want to compare cluster to cluster or groups of clusters between each genotype, instead of merging the three datasets together and then processing and clustering them
